airbyte-webapp/src/components/connectorBuilder/StreamTestingPanel/PageDisplay.tsx

```diff
@@ -58,22 +72,41 @@ export const PageDisplay: React.FC<PageDisplayProps> = ({ page, className }) =>
   return (
     <div className={classNames(className)}>
       <Tab.Group defaultIndex={defaultTabIndex}>
-        <Tab.List className={styles.tabList}>
-          {tabs.map((tab) => (
-            <Tab className={styles.tab} key={tab.key}>
-              {({ selected }) => (
-                <Text className={classNames(styles.tabTitle, { [styles.selected]: selected })}>{tab.title}</Text>
-              )}
-            </Tab>
-          ))}
-        </Tab.List>
-        <Tab.Panels className={styles.tabPanelContainer}>
-          {tabs.map((tab) => (
-            <Tab.Panel className={styles.tabPanel} key={tab.key}>
-              <pre>{tab.content}</pre>
-            </Tab.Panel>
-          ))}
-        </Tab.Panels>
+        <FlexContainer direction="column">
```

Comment:
One more thing: I'm seeing this error in console when I click the `Test` button after a refresh:
```
react_devtools_backend.js:4012 Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>.
    at div
    at FlexContainer (http://localhost:3000/static/js/src_components_ui_Flex_index_ts.chunk.js:34:5)
    at p
    at _c (http://localhost:3000/static/js/src_components_common_ApiErrorBoundary_index_ts-src_core_ApiServices_tsx-src_core_i18n_index_-3cd912.chunk.js:5615:5)
    at button
    at http://localhost:3000/static/js/vendors-node_modules_monaco-editor_react_lib_es_index_js-node_modules_lodash_debounce_js-node-86ed65.chunk.js:150863:89
    at div
    at http://localhost:3000/static/js/vendors-node_modules_monaco-editor_react_lib_es_index_js-node_modules_lodash_debounce_js-node-86ed65.chunk.js:150844:7
    at div
    at FlexContainer (http://localhost:3000/static/js/src_components_ui_Flex_index_ts.chunk.js:34:5)
    at http://localhost:3000/static/js/vendors-node_modules_monaco-editor_react_lib_es_index_js-node_modules_lodash_debounce_js-node-86ed65.chunk.js:150743:19
    at div
    at PageDisplay (http://localhost:3000/static/js/src_pages_ConnectorBuilderPage_ConnectorBuilderPage_tsx.chunk.js:6654:5)
    at div
    at ResultDisplay (http://localhost:3000/static/js/src_pages_ConnectorBuilderPage_ConnectorBuilderPage_tsx.chunk.js:6905:5)
```

Reply:
Should be gone, `Text` component was rendering a p tag.